lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
Ok so the thing that worries me about this one is that if you look at the specific commit in question, it pretty much re-indents some code and adds this line.  It's a single commit which suggests it was there to silence warnings in logs.  Reviewing the code, I wonder if the fix was correct.  I don't see $csettings used again in the scope, so maybe that's the real cause of the warning that was silenced.  It's possible that something changed at one point and the warning came about because of another change and the warning was turned off.

So this is the sort of thing where manual testing might be a really good idea because maybe the wrong problem was fixed.